ch03.asciidoc

```diff
@@ -425,12 +424,16 @@ On the other hand, this makes development more difficult as one cannot easily mo
 ** locally setting up and settling HTLCs (only peers involved)
 
 === Thoughts about Trust
-As long as a person follows the protocol and has their node secured, there is no principle risk of losing funds when participating in the Lightning Network.
-However there is the risk of paying fees when opening a channel.
-Any risk should come with a potential reward.
-In our case the reward is that she can send and receive payments of bitcoin on the Lightning Network at any time and that she can earn bitcoin by forwarding other payments.
-Alice decides that the reward is enough for her to take on the fee risk.
-Since Bob can directly close the channel, which costs fees paid by Alice, she will need to have a little trust in Bob.
+As long as a person follows the protocol and has their node secured, there is no major risk of losing funds when participating in the Lightning Network.
+However, there is the cost of paying on-chain fees when opening a channel.
+Any cost should come with a corresponding benefit.
+In our case the reward for Alice for bearing the cost of opening a chennel is that Alice can send and receive payments of bitcoin on the Lightning Network at any time and that she can earn fees in bitcoin by forwarding payments for other people.
+Alice knows that in theory Bob can close the channel immediately after opening resulting in on-chain closing fees for Alice. 
+Alice will need to have a small amount of trust in Bob. 
+Alice has been to Bob's Cafe and clearly Bob is interested in selling her coffee, so Alice can trust Bob in this sense. 
+There are mutual benefits to both Alice and Bob.
+Alice decides that the reward is enough for her to take on the cost of the on-chain fee for creating a channel to Bob. 
+In contrast, Alice will not open a channel to someone unknown in Nigeria who just sent her an uninvitedly email asking her to open a channel to him. 
```

Comment:
why Nigeria? I assume as a hint to he nigerian prince spam / scam mail. but I feel not comfortable making this trust issue about Nigeria

Reply:
Yes, it was because the Nigerian scams are the best known ones.
Yes, I also agree that it is offending to Nigeria and that it is better to remove it.
Removed in Commit # 2 in 4819aaa